### DISCUSSION TITLE
Add Fluent Store

### DISCUSSION THREAD

**Discussion by upintheairsheep** [IviriusCommunity/Rebound#98] [Ideas]
https://github.com/yoshiask/FluentStore - Fluent Store is a GUI for WinGet for end users, unlike UniGet UI, which slowly became more of a GUI for developers as time went on. It also supports a few other sources

**Comment by Lamparter** [comment]
Let's talk about this in a discussion.

**Comment by IviriusMain** [comment]
I was gonna add a Winget GUI in Rebound Control Panel just for updates

**Reply by Lamparter** [reply]
App updates don't belong in Control Panel. I think Fluent Store is a good idea, but it's quite new and I'm unsure whether using it in Rebound is the best way to go.

**Reply by upintheairsheep** [reply]
Note that Rebound 11 is intended to be for both your grandpa and experts at programming.

**Reply by IviriusMain** [reply]
> Note that Rebound 11 is intended to be for both your grandpa and experts at programming. 

Fair point, but there's also people who complain about there being a store in Windows in the first place.
I think the best way to decide is to make a poll in the Ivirius Community Discord server.

**Reply by IviriusMain** [reply]
Yoshiask said yes

**Comment by IviriusMain** [comment]
Added in v0.0.10 👀
